secp256k1zkp/depend/secp256k1-zkp/sage/generators.sage

```diff
@@ -0,0 +1,34 @@
+import sys
+import hashlib
+from sage.all import *
+
+for t in xrange(1023, -1, -1):
+	p = 2**256 -2**32 - t
+	if p.is_prime():
+		print '%x'%p
+		break
+a = 0
+b = 7
+F = FiniteField(p)
+
+o = '%x' % (EllipticCurve ([F (a), F(b)]).order())
+print o 
+
+gen2 =  hashlib.sha256('0479be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8'.decode('hex'))
+gH = EllipticCurve ([F (0), F(7)]).lift_x(int(gen2.hexdigest(),16))
+print 'gH: %s' % gH
+gen_h = '%x %x'%gH.xy()
```

Comment:
Doesn't look like this is used anywhere, are you keeping it just for documentation/context?

Reply:
This is just for documentation/reference. The point is supposed to be provably NUMS so I thought it best to have the code that generates it there for reference/review by any interested parties.